examples/features/standard/large-message/src/main/resources/jndi.properties

```diff
@@ -16,5 +16,5 @@
 # under the License.
 
 java.naming.factory.initial=org.apache.activemq.artemis.jndi.ActiveMQInitialContextFactory
-connectionFactory.ConnectionFactory=tcp://localhost:61616
+connectionFactory.ConnectionFactory=tcp://localhost:61616?minLargeMessageSize=10240
```

Comment:
It was an oversight for sure.. nice catch...

Why you set minLargeMessagesize as 10K? the default is 100K.

Reply:
ahhh... I was looking for something in the readme for the 10K and missed that... 

and you believe I wrote this years ago...  :)


merging it!

Reply:
Thanks @clebertsuconic  :)